pep-0480.txt

```diff
@@ -236,56 +258,63 @@ End-to-End Signing
 ==================
 
 End-to-end signing allows both PyPI and developers to sign for the metadata
-downloaded by clients.  PyPI is trusted to make uploaded projects available to
-clients (PyPI signs the metadata for this part of the process), and developers
-sign the distributions that they upload to PyPI.
-
-In order to delegate trust to a project, developers are required to submit a
-public key to PyPI.  PyPI takes the project's public key and adds it to parent
-metadata that PyPI then signs.  After the initial trust is established,
-developers are required to sign distributions that they upload to PyPI using
-the public key's corresponding private key.  The signed TUF metadata that
-developers upload to PyPI includes information like the distribution's file
+downloaded by clients. PyPI is trusted to make uploaded projects available to
+clients (PyPI signs for the metadata for this part of the process), and
+developers sign for the distributions that they upload to PyPI.
+
+In order to delegate trust to a project, developers are required to submit an
```

Comment:
I disagree with the "required" bit. As someone else (@joshuagl?) mentioned elsewhere in this PR, we should give developers as many options as possible for managing their keys. Sometimes, it can be with Fulcio, and sometimes it can be with GPG, and so on.